src/plugins/blocksmith/components/article-block-modal/CustomizeOptions.jsx

```diff
@@ -28,6 +30,7 @@ const CustomizeOptions = ({
   onToggleAssetPickerModal: handleToggleAssetPickerModal,
   showImage,
   updatedDate,
+  aside,
```

Comment:
@nanetteCodes alpha sort all the things pls - destructuring, props, prop types, etc. there are plugins to do this automatically to selected lines of text